callstats-java-sdk/src/main/java/io/callstats/sdk/internal/CallStatsBridgeKeepAliveManager.java

```diff
@@ -73,17 +75,23 @@ public CallStatsBridgeKeepAliveManager(int appId, String bridgeId, String token,
 		Properties prop = new Properties();
 		InputStream input = null;
 
-		input = getClass().getClassLoader().getResourceAsStream(CallStatsConst.CallStatsJavaSDKPropertyFileName);
-		if (input != null) {
-			try {
-				prop.load(input);
-				keepAliveInterval = Integer.parseInt(prop.getProperty("CallStats.keepAliveInterval"));
-			} catch (IOException e) {
-				// TODO Auto-generated catch block
-				e.printStackTrace();
+		//input = getClass().getClassLoader().getResourceAsStream(CallStatsConst.CallStatsJavaSDKPropertyFileName);
+		try {
+			input = new FileInputStream(CallStatsConst.CallStatsJavaSDKPropertyFileName);
+			if (input != null) {
+				try {
+					prop.load(input);
+					keepAliveInterval = Integer.parseInt(prop.getProperty("CallStats.keepAliveInterval"));
+				} catch (IOException e) {
+					// TODO Auto-generated catch block
+					e.printStackTrace();
+				}
 			}
+		} catch (FileNotFoundException e1) {
+			// TODO Auto-generated catch block
```

Comment:
1. do we need two level of try...catch here?
2. If exception happened here, I wonder if we should fail fast, e.g. raise the exception up in the stack instead of printing some traces and move on.